config/ROOT.scons

```diff
@@ -22,13 +22,28 @@ rootdict = Builder(generator = generate_action,
                    src_suffix = '.hh')
 ####
 
+import subprocess
+
+desired = ['python','minuit2']
```

Comment:
Looks great to me as well. Just for educational purposes, in SNO+ RAT what I had done was using ```root-config --features``` and then search the returned list for ```minuit2```.  This is more elegant, though.